README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+
+## ABOUT ME
```

Comment:
Nice use of a header here!  It looks like maybe the other headers didn't come out in the same way this one did - I think you may have to include a space between the "#" and the next letter in order for it to appear enlarged, like this one.

Reply:
@kristinawald correct, you need a space between the `#` symbol and the text in the header